config/txgh.yml

```diff
@@ -1,15 +0,0 @@
-txgh:
-    github:
-        repos:
-            matthewjackowski/txgh-test-resources:
-                api_username: <%= ENV['GITHUB_USERNAME']  %>
-                api_token: <%= ENV['GITHUB_TOKEN'] %>
-                push_source_to: <%= ENV['GITHUB_PUSH_SOURCE_TO'] %>
-                branch: <%= ENV['GITHUB_BRANCH'] %>
-    transifex:
-        projects:
-            txgh-test-1:
-                tx_config: <%= ENV['TX_CONFIG_PATH'] %>
-                api_username: <%= ENV['TX_USERNAME'] %>
-                api_password: <%= ENV['TX_PASSWORD'] %>
-                push_translations_to: <%= ENV['TX_PUSH_TRANSLATIONS_TO'] %>
```

Comment:
We should talk about how we envision multiple organizations using txgh at some point. We could turn the project into a gem for example, and require that the user provide paths to the configuration files. Thoughts?